### DISCUSSION TITLE
Unable to load extension

### DISCUSSION THREAD

**Discussion by sanegaming** [community/community#14299] [Copilot Conversations] [answered]
I am attempting to use CoPilot in Code installed on Linux from the AUR. 
CoPilot version 1.7.4016 is installed.
I get the error message "Github Copilot requires VS Code 1.57+"

I have verified that I am running Code 1.66.0-02

**Comment by davecheney** [comment, marked as answer]
Copilot only works with VSCode at the moment. VSCodium or OSS Code do not work.

**Reply by sanegaming** [reply]
Thanks, is there a plan to add support?
